packages/pods-core/src/utils.ts

```diff
@@ -211,7 +198,10 @@ export class PodUtils {
   }
 
   static hasRequiredOpts(_pClassEntry: PodClassEntryV4): boolean {
```

Comment:
can you explain the logic here? we should not show a config prompt if the pod has no required ops. instead of doing a special case, lets change this to return true if any of the fields in the pod are nullable and don't have a default